tests/integration/vlan_test.py

```diff
@@ -74,59 +74,54 @@ def test_add_and_remove_two_vlans_on_same_iface(eth1_up):
 
 
 def test_two_vlans_on_eth1_change_mtu(eth1_up):
-    desired_state = statelib.show_only(("eth1",))
-    eth1_state = desired_state[Interface.KEY][0]
-    vlans_state = create_two_vlans_state()
-    desired_state[Interface.KEY].extend(vlans_state[Interface.KEY])
-    for iface in desired_state[Interface.KEY]:
-        iface[Interface.MTU] = 2000
-    libnmstate.apply(desired_state)
+    with two_vlans_on_eth1() as desired_state:
+        desired_state[Interface.KEY].extend(eth1_up[Interface.KEY])
```

Comment:
You can extract `eth1_up[Interface.KEY][0]` to a variable and reuse it on line 88.
Then, use here `append` instead.

Same for the other tests.

Reply:
Done, thanks!